### ISSUE TITLE
Push the patches here back to upstream pipx?

### ISSUE THREAD

**Issue by pfmoore** [uranusjr/pipx-standalone#4]
I've taken the patch you made to pipx, and converted it into something that could be submitted back to the pipx project for inclusion in the main code. See https://github.com/pfmoore/pipx/tree/embedded_dist_support

Would you be OK with me proposing this as a PR for pipx itself? It would probably make maintaining this project very slightly simpler, as you wouldn't need to maintain the custom patched version (and it seems like a reasonable thing to support in pipx anyway).

**Comment by uranusjr** [comment]
Definitely. My past discussions with other pipx developers about the Windows situation didn’t go very well (e.g. why don’t you just depend on Scoop’s Python instead), but that’s a while ago now and you may have better luck 🙂

**Comment by pfmoore** [comment]
I finally got round to submitting https://github.com/pipxproject/pipx/pull/465
